_posts/12-05-01-Building-your-Application.md

```diff
@@ -23,34 +23,34 @@ Među zadacima koje biste možda želeli da automatizujete su:
-zgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
+Alati za izgradnju (build) mogu biti shvaćeni kao skup skripti koje se bave određnim učestalim zadacima isporuke (deploy-a) softvera. 
+Alat za izgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
 
-Postoji mnogo dostupnih alata otvorenog koda da vam pomogne sa automatizacijom, neki su napisani u PHP-u, drugi nisu.
-To ne treba da vas sputava da ih koristite, ako više odgovaraju za određeni posao. Evo nekoliko primera:
+Postoji puno alata otvorenog koda koji vam mogu pomoći pri automatizovanoj izgradnji (build), neki su napisani u PHP-u ali 
+postoje oni koji nisu. To ne treba da vas sputava da ih koristite, ako više odgovaraju za određeni posao. Evo nekoliko primera:
 
-[Phing](http://www.phing.info/) je najlakši način da počnete sa primenom automatizacije isporuivanja koda u svetu PHP-a. Pomoću Phing-a
-možete da kontrolišete pakovanje, angažovanje(deployment) ili testiranje procesa unutar jednostavne XML build datoteke. Phing (koji se zasniva na [Apache Ant](http://ant.apache.org/)) obezbeđuje bogat set zadataka, obično potrebnih za instalaciju ili ažuriranje web aplikacija i može se proširiti dodatnim prilagođenim zadacima, pisanim u PHP-u.
+[Phing](http://www.phing.info/) je najlakši način da počnete sa primenom automatizacije isporučivanja koda u svetu PHP-a. Pomoću Phing-a
+možete da kontrolišete pakovanje, isporuku ili testiranje procesa unutar jednostavne XML build datoteke. Phing (koji se zasniva na [Apache Ant](http://ant.apache.org/)) obezbeđuje bogat set zadataka, obično potrebnih za instalaciju ili ažuriranje web aplikacija i može se proširiti dodatnim prilagođenim zadacima, pisanim u PHP-u.
 
 [Capistrano](https://github.com/capistrano/capistrano/wiki) je sistem namenjen *srednjim-do-naprednim programerima* za
-izvršavanje komandi na struktuiran, ponovljivi način na jednoj ili više udaljenih mašina. Prekonfigurisan je za primenu (deploying) na Ruby on Rails applikacijama, međutim ljudi **uspešno angažuju(deploying) PHP sisteme** sa njim. Uspešno korišćenje Capistrana zavisi od radnog iskustva sa Ruby i Rake.
+izvršavanje komandi na struktuiran, ponovljivi način na jednoj ili više udaljenih mašina. Prekonfigurisan je za primenu na Ruby on Rails applikacijama, međutim ljudi **uspešno isporučuju PHP sisteme** sa njim. Uspešno korišćenje Capistrana zavisi od radnog iskustva sa Ruby i Rake.
 
 Blog post Dejva Gardnera [PHP Deployment with Capistrano](http://www.davegardner.me.uk/blog/2012/02/13/php-deployment-with-capistrano/)
 je dobra polazna tačka za PHP developere zainteresovane za Capistrano.
 
 [Chef](http://www.opscode.com/chef/) je više od skeleta(framework) za primenu(deployment), to je veoma moćan Ruby baziran sistemski integracioni skelet(framework) koji ne samo da razvija(deploy) Vaše aplikacije već može da izgradi i celokupno serversko okruženje ili virtuelne kutije(virtual boxes).

```

Comment:
Mi smo do sada kao alternativu za *framework* koristili samo izraz *radni okvir* (mada smo i to izbegavali), nikada *skelet*, a ja bih u ovom konkretnom slučaju čak ostavio framework, ne znam. Šta ti misliš?

Tu opet ima ono *primena/deployment*, kao i *razvija/deploy*. Meni to "primena" i "razvoj" nikako nema smisla kada je reč *deploy* u pitanju.